packages/terra-application-utility/src/utility/_UtilityMenu.jsx

```diff
@@ -41,6 +41,11 @@ const propTypes = {
    * This will be provided by the terra-application-header or terra-application-menu.
    */
   onRequestClose: PropTypes.func,
+  /**
+   *
+   * The role attribute of Utility Menu.
+   */
+  menurole: PropTypes.string,
```

Comment:
change this to `menuRole` in order for the jest test to pass
```suggestion
  menuRole: PropTypes.string,
```

Reply:
[propName has been changed to camelcase](https://github.com/cerner/terra-framework/pull/501/commits/8e717f363590ac70e52d300484c4a414037191d9) in all places except for utility button since react does not recognize the prop with camel case on DOM Element

Reply:
What's the error React throws? UtilityButton renders the UtilityMenu, not a native DOM element - I'd think we'd be fine passing that prop

Reply:
it throws below warning when I change prop on utility button to menuRole : <img width="1680" alt="screen shot 2019-03-04 at 9 54 17 pm" src="https://user-images.githubusercontent.com/12869809/53746918-2a906c00-3ec8-11e9-9dba-a2a635920d08.png"> .

Reply:
We need to destructure the `menuRole` prop from `this.props`. I added a comment on UtilityButton. Make that change and change `menurole` to `menuRole` for the higher order components - does this resolve the issue?

Reply:
It looks like we spread all custom props onto the utility button resulting in `menuRole` being applied as a prop / attribute on the DOM element. We likely need some logic to delete menuRole out of custom props so it is not added to the DOM element.

It looks like in `packages/terra-application-utility/src/utility/_UtilityButton.jsx` on line 101, we `delete customProps.onClick`. We should test doing the same thing for menuRole on line 102.
Add the following:
`delete customProps.menuRole;`

Reply:
this worked..! thanks for suggestion @bjankord [prop has been renamed to camel case](8bb27390433900a5998d80ec415a7d74b3e59624)